doc/replication.md

```diff
@@ -98,8 +98,12 @@ CDTReplicator *replicator = [replicatorFactory oneWay:pushReplication error:&err
 
 // Check replicator isn't nil, if so check error
 
-// Start the replication and wait for it to complete
-[replicator start];
+// Start the replication
+[replicator startWithError:&error];
+
+//check replication started
```

Comment:
done in cfcfb00 however I didn't spell out the best way to handle the error as that would be down to their application's business logic.